src/probnum/random_variables/_dirac.py

```diff
@@ -73,7 +73,7 @@ def __init__(
             sample=self._sample,
             in_support=lambda x: np.all(x == self._support),
             pmf=lambda x: np.float_(1.0 if np.all(x == self._support) else 0.0),
-            cdf=lambda x: np.float_(0.0 if np.any(x < self._support) else 0.0),
+            cdf=lambda x: np.float_(1.0 if np.all(x >= self._support) else 0.0),
```

Comment:
Super petty point: your original formulation (with 1.0 corrected), i.e.
```python
cdf=lambda x: np.float_(0.0 if np.any(x < self._support) else 1.0)
```
might be faster on average since `np.any` can exit without checking all elements of the boolean input array while `np.all` has to check all elements. I understand that this way it is more readable. I'll leave it to you what you prefer.